### ISSUE TITLE
Background checkerboard for transparent pictures

### ISSUE THREAD

**Issue by sandebert** [jurplel/qView#489]
It would be nice if we could have a checkerboard background for pictures with transparancy.

Example:
![image](https://user-images.githubusercontent.com/3855726/160376653-2961f8fa-c804-42e7-bbd1-3b74d81c94c6.png)

**Comment by MrGit3** [comment]
I agree. There is currently no way to differentiate between transparency and the end of the image.

**Comment by jdpurcell** [comment]
@jurplel In case you were considering this feature for qView, I implemented it in my fork but took a different approach than #494. Personally I don't plan to use it, but I implemented it just kind of for fun.
